prog/vnet/update_firewall_rules.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+class Prog::Vnet::UpdateFirewallRules < Prog::Base
+  subject_is :vm
+
+  label def update_firewall_rules
+    rules = vm.private_subnets.map(&:firewall_rules).flatten
+    allowed_ingress_ip4 = rules.select { !_1.ip6? }.map { _1.ip.to_s }
+    allowed_ingress_ip6 = rules.select { _1.ip6? }.map { _1.ip.to_s }
+    vm.vm_host.sshable.cmd(<<FLUSH_RULES)
+sudo ip netns exec #{vm.inhost_name} bash -c 'sudo nft flush set inet fw_table allowed_ipv4_ips && sudo nft flush set inet fw_table allowed_ipv6_ips'
+FLUSH_RULES
+
+    vm.vm_host.sshable.cmd(<<ADD_RULES)
+sudo ip netns exec #{vm.inhost_name} bash -c 'sudo nft --file - <<EOF
```

Comment:
Why do this in a bash heredoc rather than `cmd("sudo ip netns exec #{vm.inhost_name} nft --file -", stdin: str)`? It seems cleaner that way, it's rather odd to have a ginormous argv if you can avoid it.